libkrb5/src/context.rs

```diff
@@ -300,6 +300,8 @@ impl Krb5Context {
         let code = unsafe {krb5_auth_con_set_req_cksumtype(self.context, auth_context.auth_context, GSS_CHECKSUM_TYPE)};
         krb5_error_code_escape_hatch(self, code)?;
 
+        auth_context.set_flags(Krb5AuthContextOptions::Replay as i32 | Krb5AuthContextOptions::Sequence as i32)?;
```

Comment:
Ha jól értem mi a szerver felé mindig kérni fogunk sequence numbert az AP_REQ-ben, de a kliens ezt saját hatáskörben vagy kéri, vagy nem? A replay pedig a sequencel kéz a kézben jár, hogy ne lehessen repeated message... A sequence number meglétét ellenőrizzük, de a repeated messageket nem kell, mert az mitkrb lib kezeli?

Reply:
Igen, a kliens dönti el, hogy legyen-e sequence number. (Jelenleg mi ezt mindig kérjük a szerver fele). 
Kód alapán úgy látom a repeated messageket a lib fogja ellenőrizni, ha az auth contextben be van állítva a `KRB5_AUTH_CONTEXT_DO_TIME` flag. 
Ez a sor amúgy problémás volt, mert rossz flageket használtam, ezt javítottam most.